nucliadb/nucliadb/ingest/orm/resource.py

```diff
@@ -129,6 +129,10 @@
     FieldText.Format.JSON: "application/json",
 }
 
+BASIC_IMMUTABLE_FIELDS = [
```

Comment:
nit: if this `list` is constant and not going to be changed by some code at runtime, use a `tuple` instead